platform/ext/target/nordic_nrf/common/gcc/nordic_nrf_s.ld.template

```diff
@@ -0,0 +1,691 @@
+        {{pattern}}(.text*)
+        {{pattern}}(.rodata*)
+        {% endfor %}
+    {% endif %}
+        *({{manifest.manifest.name}}_ATTR_FN)
+        . = ALIGN(32);
+    } > FLASH
+    Image$${{manifest.manifest.name}}_LINKER$$RO$$Base = ADDR(.{{manifest.manifest.name}}_LINKER);
+    Image$${{manifest.manifest.name}}_LINKER$$RO$$Limit = ADDR(.{{manifest.manifest.name}}_LINKER) + SIZEOF(.{{manifest.manifest.name}}_LINKER);
+    Image$${{manifest.manifest.name}}_LINKER$$Base = ADDR(.{{manifest.manifest.name}}_LINKER);
+    Image$${{manifest.manifest.name}}_LINKER$$Limit = ADDR(.{{manifest.manifest.name}}_LINKER) + SIZEOF(.{{manifest.manifest.name}}_LINKER);
+    {% if manifest.attr.conditional %}
+#endif /* {{manifest.attr.conditional}} */
+    {% endif %}
+
+    {% endif %}
+{% endfor %}
+    /**** APPLICATION RoT RO part (CODE + RODATA) end here */
+    Image$$TFM_APP_CODE_END$$Base = .;
+
+#if defined(S_CODE_SRAM_ALIAS_BASE)
+    .ER_CODE_SRAM : ALIGN(4)
+    {
+        *Driver_GFC100_EFlash.o(.text*)
+        *Driver_GFC100_EFlash.o(.rodata*)
+        *gfc100_eflash_drv.o(.text*)
+        *gfc100_eflash_drv.o(.rodata*)
+        *musca_b1_eflash_drv.o(.text*)
+        *musca_b1_eflash_drv.o(.rodata*)
+        . = ALIGN(4); /* This alignment is needed to make the section size 4 bytes aligned */
+    } > CODE_RAM AT > FLASH
+    Image$$ER_CODE_SRAM$$RO$$Base = ADDR(.ER_CODE_SRAM);
+    Image$$ER_CODE_SRAM$$RO$$Limit = ADDR(.ER_CODE_SRAM) + SIZEOF(.ER_CODE_SRAM);
+    Image$$ER_CODE_SRAM$$Base = ADDR(.ER_CODE_SRAM);
+    Image$$ER_CODE_SRAM$$Limit = ADDR(.ER_CODE_SRAM) + SIZEOF(.ER_CODE_SRAM);
+#endif
+
+#if TFM_LVL != 1
+    .ARM.extab :
+    {
+        *(.ARM.extab* .gnu.linkonce.armextab.*)
+    } > FLASH
+
+    __exidx_start = .;
+    .ARM.exidx :
+    {
+        *(.ARM.exidx* .gnu.linkonce.armexidx.*)
+    } > FLASH
+    __exidx_end = .;
+
+#endif /* TFM_LVL != 1 */
+
+    .ER_TFM_CODE : ALIGN(4)
+    {
+        *(.text*)
+
+        KEEP(*(.init))
+        KEEP(*(.fini))
+
+
+        /* .ctors */
+        *crtbegin.o(.ctors)
+        *crtbegin?.o(.ctors)
+        *(EXCLUDE_FILE(*crtend?.o *crtend.o) .ctors)
+        *(SORT(.ctors.*))
+        *(.ctors)
+
+        /* .dtors */
+         *crtbegin.o(.dtors)
+         *crtbegin?.o(.dtors)
+         *(EXCLUDE_FILE(*crtend?.o *crtend.o) .dtors)
+         *(SORT(.dtors.*))
+         *(.dtors)
+    } > FLASH
+
+#ifndef TFM_MULTI_CORE_TOPOLOGY
+    /*
+     * Place the CMSE Veneers (containing the SG instructions) in a correctly
+     * aligned region so that the SAU can programmed to just set this region as
+     * Non-Secure Callable.
+     */
+    .gnu.sgstubs : ALIGN(32)
+    {
+        . = CMSE_VENEER_REGION_START_ALIGN;
+        Load$$LR$$LR_VENEER$$Base = ABSOLUTE(.);
+        *(.gnu.sgstubs*)
+#ifdef CMSE_VENEER_REGION_IN_CODE
+    } > FLASH
+#else
+    } > VENEERS
+#endif
+
+    /*
+     * This is needed because the linker seems to place the veneers *after* the
```

Comment:
talking about _this_ part here

Reply:
It's sort of the same change, but here it is used in our linker script, but the upstream PR is for the tfm_common_s.ld.

Reply:
yes of course.